Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest/ContestDetailsPage.tsx

```diff
@@ -152,45 +176,53 @@ const ContestDetailsPage = () => {
                         </>
                     )
                     : null}
-                <LinkButton
-                  id="button-card-compete"
-                  internalClassName={competableOnlyForAdminAndLecturers
-                      ? styles.adminAccessibleButton
-                      : ''}
-                  to={getParticipateInContestUrl({
-                      id: contestIdToNumber,
-                      participationType: ContestParticipationType.Compete,
-                  })}
-                  text="Compete"
-                  state={
-                      canAccessCompeteButton
-                          ? ButtonState.enabled
-                          : ButtonState.disabled
-                }
-                />
-                <LinkButton
-                  id="button-card-practice"
-                  internalClassName={praticableOnlyForAdminOrLecturers
-                      ? styles.adminAccessibleButton
-                      : ''}
-                  to={getParticipateInContestUrl({
-                      id: contestIdToNumber,
-                      participationType: ContestParticipationType.Practice,
-                  })}
-                  text="Practice"
-                  type={LinkButtonType.secondary}
-                  state={
-                      canAccessPracticeButton
-                          ? ButtonState.enabled
-                          : ButtonState.disabled
-                }
-                />
+                {canAccessCompeteButton
+                    ? (
+                        <LinkButton
+                          id="button-card-compete"
+                          internalClassName={competableOnlyForAdminAndLecturers
+                              ? styles.adminAccessibleButton
+                              : ''}
+                          to={getParticipateInContestUrl({
+                              id: contestIdToNumber,
+                              participationType: ContestParticipationType.Compete,
+                          })}
+                          text="Compete"
+                          state={
+                        canAccessCompeteButton
+                            ? ButtonState.enabled
+                            : ButtonState.disabled
+                    }
+                        />
+                    )
+                    : null}
+                {canAccessPracticeButton
```

Comment:
Space here

Reply:
There is no ESLint rule that enforces space here, so this is the way it is done everywhere in the app.